packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -531,4 +531,46 @@ void main() {
     final TextField widget = tester.widget(find.byType(TextField));
     expect(widget.selectionControls, equals(materialTextSelectionControls));
   });
+
+  testWidgets('TextFormField respects hintTextDirection', (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
```

Comment:
The entire body should indent 2 fewer spaces.
```suggestion
    await tester.pumpWidget(MaterialApp(
```